### PR TITLE
fix/updated-readme

### PR DESCRIPTION
**Pull request by aberonja**
Updated readme file with new instructions to setup project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ These instructions will get you a copy of the project up and running on your loc
 #### Prerequisites
 What things you need to setup before starting server
 
-- Install [Java 10](https://docs.oracle.com/javase/10/install/overview-jdk-10-and-jre-10-installation.htm#JSJIG-GUID-8677A77F-231A-40F7-98B9-1FD0B48C346A) and set the JAVA_HOME Environment Variable
+- Install [Java 10](https://docs.oracle.com/javase/10/install/overview-jdk-10-and-jre-10-installation.htm#JSJIG-GUID-8677A77F-231A-40F7-98B9-1FD0B48C346A) ([Windows](https://drive.google.com/file/d/1EE0cBbfHbFgB-zxBx_nAnORO19HCZCkB/view?usp=sharing)) and set the JAVA_HOME Environment Variable. Java 10 is mandatory because newer versions of Java are not going to work.
```
Comment (iscekic) [inline README.md]: Why are we using drive links here? Also the `oracle` link can be shorter, without the `guid`.

File: README.md [hunk 2]
```diff
@@ -53,6 +53,10 @@ project_root
 mvn clean install
 ```
 
+4 . Run SQL [script](https://drive.google.com/file/d/1Uc1VmwZsHL1_7q25AIAXiwWSw8WWWGVg/view?usp=sharing) to populate database
```
Comment (iscekic) [inline README.md]: Shouldn't this script be part of the repo?

File: README.md [hunk 3]
```diff
@@ -53,6 +53,10 @@ project_root
...
+
+5 . Add you credentials with **@execom** domain to the user table in order to login into the app
```
Comment (iscekic) [inline README.md]: Write an example of how to do this.

File: README.md (hunk 1 above)
Comment (aberonja) [reply README.md]: We're using drive because there is no official link for Java 10 and that's why we downloaded it and placed it on the drive.

Comment (iscekic) [reply README.md]: Please make sure the link is accessible to everyone in execom. 🕺

Comment (aberonja) [reply README.md]: Already done it 👍

File: README.md (hunk 3 above)
Comment (aberonja) [reply README.md]: Added gifs for script running and adding credentials.

**Pull request merged**